tests/tests.js

```diff
@@ -67,7 +93,7 @@ describe('parsing', () => {
   });
 
   it('handles components with properties and expressions', () => {
-    const result = unwrap(html`<${Foo} a=${1}/>${2}`);
+    const result = unwrap(html`<${Foo} a=${1}/>${2}<//>`);
```

Comment:
```
<${Foo} a=${1}/>${2}<//>
              ^
```

This one already is self closing, does this test even need any changes?

Reply:
ah sorry, my brain tricked me, one sec 👍